src/Microsoft.AspNetCore.Razor.Tools/GenerateCommand.cs

```diff
@@ -27,6 +27,7 @@ public GenerateCommand(Application parent)
             Configuration = Option("-c", "Razor configuration name", CommandOptionType.SingleValue);
             ExtensionNames = Option("-n", "extension name", CommandOptionType.MultipleValue);
             ExtensionFilePaths = Option("-e", "extension file path", CommandOptionType.MultipleValue);
+            GenerateDeclaration = Option("-g", "Generate declaration", CommandOptionType.NoValue);
```

Comment:
possible to use `--generate-declarations` or something like that? I regret making these options so terse.